maxim/logger/livekit/agent_session.py

```
@@ -1,15 +1,26 @@
 import functools
 import inspect
 import traceback
+import time
+from typing import List 
```

Ruff (0.12.2)

`typing.List` is deprecated, use `list` instead

(UP035)